cmd/print.go

```diff
@@ -23,12 +23,23 @@ import (
 	"github.com/spf13/cobra"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/cli-runtime/pkg/printers"
+	"k8s.io/client-go/tools/clientcmd"
 )
 
 var (
 	// outputFormat contains currently set output format. Value assigned via --output/-o flag.
 	// Defaults to YAML.
 	outputFormat = "yaml"
+
+	// namespace contains currently set namespace that should be used. Value assigned via
+	// --namespace/-n flag.
+	// Default behavior is to use the current namespace the user is in.
+	namespace string
+
+	// allNamespaces indicates whether all namespaces should be used. Value assigned via
+	// --all-namespaces/-A flag.
+	// If present, overrides the namespace variable.
```

Comment:
```suggestion
```

Reply:
removing this in favour of the bottom comment about `MarkFlagsMutuallyExclusive`